.github/workflows/finalize-release.yaml

```diff
@@ -29,8 +29,7 @@ jobs:
           git config --local user.name imodeljs-admin
       - name: Run update-changelogs.mjs
         run: |
-          npm i -g zx
-          npm install -g @microsoft/rush
-          zx --install ./.github/workflows/automation-scripts/update-changelogs.mjs
+          sudo npm install -g @microsoft/rush
```

Comment:
Why `sudo`? Did this ever work before without it?

Reply:
it did, not sure why it started failing since last minor release. @hl662 were you able to figure out what caused this?

Reply:
No, the difference between the failed job and the last successful release was the github runner's version. Comparing the versions between the two, nothing jumped out to me that suggested additional file permissions were needed

Reply:
Can we remove the sudo then?

Reply:
No, removing the sudo would cause it to throw a permissions error. Why it never failed before us introducing `sudo` is still unclear. Github could have update their *nix agents to require extra permissions to access+write to `root` folders (which is what happens when we do `install -g`). As I'm writing this down I'm surprised why it never failed to begin with... Seems to me like throwing the permissions error should have been the right behavior from the beginning.

Reply:
I see. Perhaps changing the npm default dir?
https://docs.npmjs.com/resolving-eacces-permissions-errors-when-installing-packages-globally

https://github.com/actions/setup-node also seems to have some success (it changes the location for you, I believe)

Reply:
I can look into this in a separate PR, maybe see if we can avoid having to install rush globally, but I don't want to delay the finalize release workflow further